### PR TITLE
1st draft of 8.2 what's new

### PR DESCRIPTION
**Pull request by ollyhowell**
Closes https://github.com/elastic/obs-docs-projects/issues/47

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
 
 // What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
 
+[discrete]
+== AWS serverless log ingestion for faster insight
+
+The elastic-serverless-forwarder application enables you to ingest logs from the
+AWS environment into Elastic in a complete serverless fashion. You don’t need to
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:19]: ?

```suggestion
AWS environment into Elastic in a completely serverless fashion. You don’t need to
```

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+set up and maintain compute resources, which simplifies the getting started
+experience and reduces data onboarding friction. The application is publicly
+available in Serverless Application repository (SAR), an AWS-managed repository
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:22]:
```suggestion
available in the Serverless Application repository (SAR), an AWS-managed repository
```

File: docs/en/observability/whats-new.asciidoc [hunk 3]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+of {kib} to analyze the collected data.
+
+[role="screenshot"]
+image::images/integrations-custom-aws-logs.png[Add Customer AWS Logs integration]
+
+[discrete]
+== {agent} Azure virtual machine extension
+
+The new Microsoft Azure VM extension for {agent} provides Azure users a
+very simple way to install and register {agent} on all their Azure VM’s.
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:59]:
```suggestion
straightforward way to install and register {agent} on all their Azure VMs.
```

File: docs/en/observability/whats-new.asciidoc [hunk 4]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+It automatically installs {agent}, registers it with {fleet}, creates a
+separate policy that collects system logs and metrics, and assigns this policy
+— all with a single checkbox or command line option.
+
+You can use the Elastic deployment pane on the Azure console to configure
+Elastic Azure VM extension with a single click. If you prefer to manage with
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:65]:
```suggestion
the Elastic Azure VM extension with a single click. If you prefer to manage with
```

File: docs/en/observability/whats-new.asciidoc [hunk 5]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+scripts, you can use Azure CLI or Powershell to interact with the extension and
+get {agent} installed and configured on all your Azure VM’s.
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:67]: I'm questioning this suggestion now 🤔 

```suggestion
get {agent} installed and configured on all your Azure VMs.
```

File: docs/en/observability/whats-new.asciidoc [hunk 6]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+
+
+[role="screenshot"]
+image::images/APM-service-groups.png[APM Service groups]
+
+[discrete]
+== APM visibility for AWS Lambda Functions
+
+Elastic APM now supports distributed tracing for AWS Lambda Functions running in
+production environments. This feature is currently supported by the Java, Python
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:126]:
```suggestion
production environments. This feature is currently supported by the Java, Python,
```

File: docs/en/observability/whats-new.asciidoc [hunk 7]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+and Node.js Agents. It provides performance monitoring, error monitoring, and
+topology views showing dependencies within the distributed microservices
+architecture. Additionally, auto-instrumentation of traces, transactions and
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:129]:
```suggestion
architecture. Additionally, auto-instrumentation of traces, transactions, and
```

File: docs/en/observability/whats-new.asciidoc [hunk 8]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+== Support for compressed spans in Elastic APM
+
+You can now reduce storage and processing overheads associated with low-value
+spans such as those encountered in an N+1 query to a database or cache.
+Normally, it would be quite difficult to analyze a trace waterfall in the UI
+involving hundreds of short duration spans. This feature enables such spans to
+be represented by a single aggregated span. The compressed span also provides
+contextual information such as the total number of spans included, total
+duration, and average span duration. This feature is currently supported by the
+Java, .NET, Python, GO and PHP Agents.
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:153]:
```suggestion
Java, .NET, Python, GO, and PHP Agents.
```

File: docs/en/observability/whats-new.asciidoc [hunk 9]
```diff
@@ -3,8 +3,6 @@
 
 Here are the highlights of what's new and improved in {minor-version}.
 
-coming[8.0.0]
-
 // Other versions:
 // {observability-guide-all}/8.1/whats-new.html[8.1] |
 // {observability-guide-all}/8.0/whats-new.html[8.0] |
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:8]: Don't forget to uncomment these previous versions

**Review by bmorelli25 — COMMENTED** [review]
A few suggestions

File: docs/en/observability/whats-new.asciidoc [hunk 10]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+establish a secure connection between agents and the {ls} instance. Please
+note that mutual TLS is enforced when agents are set to write to a {ls} output.
+
+[role="screenshot"]
+image::images/fleet-add-logstash-output.gif[Add new {ls} output via {fleet}]
+
+[discrete]
+== APM support for {kib} spaces
+
+You can now configure APM UI indices in the context of {kib} spaces. This
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:95]: I think "APM UI indices" is too condensed. Makes it sound like the indices are only valid when used with the UI. If @bmorelli25 thinks it's Ok as-is, I will back off.

**Review by dedemorton — COMMENTED** [review]
Just a few minor comments.

File: docs/en/observability/whats-new.asciidoc (hunk 10 above)
Comment (bmorelli25) [reply docs/en/observability/whats-new.asciidoc:95]: I think this is OK as-is. This feature allows users to customize the indices that the APM UI queries—so it is only valid when used with the UI.

File: docs/en/observability/whats-new.asciidoc [hunk 11]
```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
...
+
+You can use the Elastic deployment pane on the Azure console to configure
+Elastic Azure VM extension with a single click. If you prefer to manage with
+scripts, you can use Azure CLI or Powershell to interact with the extension and
+get {agent} installed and configured on all your Azure VM’s.
+
+[role="screenshot"]
+image::images/azure-elastic-vm.png[{es} VM extension installed on Azure]
+
+Once configured you can manage the lifecycle of your {agents} from within
```
[the remark this answers is not in the document]
Comment (bmorelli25) [reply docs/en/observability/whats-new.asciidoc:72]:
```suggestion
Then you can manage the lifecycle of your {agents} from within
```

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**